graphrag_sdk/document_loaders/csv.py

```
@@ -1,5 +1,6 @@
 import csv
 from typing import Iterator
+from typing import Optional
```

Ruff

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)